feature: Implemented Export2MONGODB pipeline

imspyde opened the pull request.

tech-engine commented:
Please make sure to close connection upon exit.

tech-engine commented:
Here is a quick peek, hope it helps.

__` Close`__
```go
//  something like the below code should go in your close hook of the pipeline
_ = client.Disconnect(<context here>)
```

__`Export2MONGODB func`__
```go 
// something like below code should go in Export2MONGODB func
if err := client.Database("admin").RunCommand(context.TODO(), bson.D{{"ping", 1}}).Err(); err != nil {
      return nil, fmt.Errorf("Export2MONGODB: error ping to DB failed %w", err)
}
```

Thanks

imspyde commented:
Thank you for your feedback. I've addressed the comments and made the necessary changes in the pipeline and Export2MONGODB function to ensure proper connection closing and error handling. Here's what I've done:

1. Added the necessary code to close the MongoDB connection upon exit in the pipeline's close hook.
2. Implemented error handling in the Export2MONGODB function, including pinging the database to detect errors and returning informative error messages.

Please review the changes and let me know if there are any further suggestions or improvements needed.

Thank you!

imspyde commented:
Thank you for your feedback. I've addressed the comments and made the necessary changes in the pipeline and Export2MONGODB function to ensure proper connection closing and error handling. Here's what I've done:

Added the necessary code to close the MongoDB connection upon exit in the pipeline's close hook.
Implemented error handling in the Export2MONGODB function, including pinging the database to detect errors and returning informative error messages.
Please review the changes and let me know if there are any further suggestions or improvements needed.

Thank you!
